File: javascript/1012.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var linha = lines.shift().split(' ');
var first = parseFloat(linha[0]);
var second = parseFloat(linha[1]);
var third = parseFloat(linha[2]);

var triangulo = (first * third / 2);
var circulo = (3.14159 * Math.pow(third, 2));
var trapezio = (((first + second) * third) / 2);
var quadrado = (Math.pow(second, 2));
var retangulo = (first * second);
console.log("TRIANGULO: " + triangulo.toFixed(3));
console.log("CIRCULO: " + circulo.toFixed(3));
console.log("TRAPEZIO: " + trapezio.toFixed(3));
console.log("QUADRADO: " + quadrado.toFixed(3));
console.log("RETANGULO: " + retangulo.toFixed(3));

File: Java/1012.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
public class Main {
    public static void main(String[] args) throws IOException {
        InputStreamReader ir = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(ir);
        String linha = in.readLine();
        String[] valores = linha.trim().split(" ");
        float a = Float.parseFloat(valores[0]);
        float b = Float.parseFloat(valores[1]);
        float c = Float.parseFloat(valores[2]);
        float triangulo = a*c/2;
        double circulo = 3.14159 * Math.pow(c,2);
        float trapezio = ((a+b)*c)/2;
        float quadrado = b*b;
        float retangulo = a*b;
	    System.out.printf("TRIANGULO: %.3f\nCIRCULO: %.3f\n",triangulo,circulo);
	    System.out.printf("TRAPEZIO: %.3f\nQUADRADO: %.3f\n",trapezio,quadrado);
	    System.out.printf("RETANGULO: %.3f\n",retangulo);
	}
}